memphis/memphis.py

```diff
@@ -34,6 +34,7 @@
 from memphis.types import Retention, Storage
 from memphis.utils import get_internal_name, random_bytes
 
+app_id = str(uuid.uuid4())
```

Comment:
Is this implemented as a module-level variable because you want it to generate once per import (vs per connection or per consumer / producer instance)?  Might be worth putting a comment here.

Reply:
Yes exactly we want to have the ability to gather producers/consumers created within the same app no matter if they share the same connection or not